sidebars.js

```diff
@@ -34,259 +34,268 @@ module.exports = {
-                },
-                {
-                  type: 'doc',
-                  id: 'zkapps/snarkyjs-reference/interfaces/ProvablePure',
-                  label: 'ProvablePure',
-                },
-              ],
-            },
-            {
-              type: 'category',
-              label: 'Modules',
-              items: [
-                {
-                  type: 'doc',
-                  id: 'zkapps/snarkyjs-reference/modules/Encoding',
-                  label: 'Encoding',
-                },
-                {
-                  type: 'doc',
-                  id: 'zkapps/snarkyjs-reference/modules/Encryption',
-                  label: 'Encryption',
-                },
-                {
-                  type: 'doc',
-                  id: 'zkapps/snarkyjs-reference/modules/Experimental',
-                  label: 'Experimental',
-                },
-                {
-                  type: 'doc',
-                  id: 'zkapps/snarkyjs-reference/modules/Mina',
-                  label: 'Mina',
-                },
-                {
-                  type: 'doc',
-                  id: 'zkapps/snarkyjs-reference/modules/Types.Json',
-                  label: 'Types.Json',
-                },
-                {
-                  type: 'doc',
-                  id: 'zkapps/snarkyjs-reference/modules/Types',
-                  label: 'Types',
-                },
-              ],
-            },
-          ],
-        },
-        'zkapps/roadmap',
-        'zkapps/faq',
-        'zkapps/zkapps-for-ethereum-developers',
+        type: 'category',
+        label: 'SnarkyJS',
+        items: [
+          'zkapps/snarkyjs/snarkyjs-overview',
+          'zkapps/snarkyjs/snarkyjs-intro',
+          'zkapps/snarkyjs/writing-circuits',
+          'zkapps/snarkyjs/writing-smart-contracts',
+          'zkapps/snarkyjs/interact-with-mina',
+          'zkapps/snarkyjs/fetch-events-and-actions',
+          'zkapps/snarkyjs/events',
+          'zkapps/snarkyjs/actions-and-reducer',
+          'zkapps/snarkyjs/on-chain-values',
+          'zkapps/snarkyjs/recursion',
+          'zkapps/snarkyjs/custom-tokens',
+          'zkapps/snarkyjs/merkle-tree',
+          'zkapps/snarkyjs/permissions',
+          'zkapps/snarkyjs/time-locked-accounts',
+        ],
+      },
+      {
+        type: 'category',
+        label: 'Tutorials',
+        items: [
+          'zkapps/tutorials/hello-world',
+          'zkapps/tutorials/private-inputs-hash-functions',
+          'zkapps/tutorials/deploying-to-a-network',
+          'zkapps/tutorials/zkapp-ui-with-react',
+          'zkapps/tutorials/common-types-and-functions',
+          'zkapps/tutorials/offchain-storage',
+          'zkapps/tutorials/oracle',
+          'zkapps/tutorials/custom-tokens',
+          'zkapps/tutorials/recursion',
+          'zkapps/tutorials/anonymous-message-board',
+          'zkapps/tutorials/interacting-with-zkapps-server-side',
+        ],
+      },
+      {
```

Comment:
I didn't change any of this content, but I did outdent to match the other part of our proposed architecture 

@mitschabaude I need eagle eyes and JavaScript training, I broke our sidebar oops